server/engine/rooms.js

```diff
@@ -25,21 +27,33 @@ class Rooms {
     return name.join("");
   }
 
-  createRoom(username) {
+  createRoom(userId) {
+    const user = users.getUser(userId);
+    if (!user) return null;
     const name = this.nextName();
-    const room = { name: name, creator: username, users: [username] };
+    const room = { name: name, users: [userId] };
```

Comment:
Any reason for removing the creator of the room? 
My thought for adding it was that we probably wanted to have a user who can act as a moderator of the group, doing things like assigning game values, kicking people, etc. Creator might not be the best name for this though.

Reply:
I was thinking of having just the user at index 0 is the moderator/leader. That's how netgames.io does it